src/components/views/dialogs/spotlight/SpotlightDialog.tsx

```diff
@@ -1066,6 +1067,43 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", initialFilter = n
         );
     }
 
+    const calculateNumOfEleInSection = (refs: RefObject<HTMLElement>): number => {
```

Comment:
The name of the method isn't readable, `refs` also seems wrong given it takes a single param not a list/array/set